tests/unit/services/file-manager-test.js

```diff
@@ -91,7 +91,7 @@ module('Unit | Service | file manager', function(hooks) {
     hooks.beforeEach(function() {
         this.owner.register('service:session', sessionStub);
     });
-    test('getContents sends valid waterbutler request', async function (assert) {
+    skip('getContents sends valid waterbutler request', async function (assert) {
```

Comment:
oh no! What happened here? Just run low on time or is it particularly troublesome?

Reply:
Tests died because I removed mockjax. But, I'm reasonably certain that we're not using the file manager service anywhere. We've replicated all of its individual file functionality directly on the file model, and that's what we use for quick files. I think we should consider removing the file-manager service in favor of methods on models.

Reply:
Yeah, if we're not using it, we should remove it. I can't think of a reason we'd be accessing the file away from the model in any case. We can make a ticket for that and leave this skipped.